app/Http/Controllers/Manager/AuthController.php

```diff
@@ -418,6 +418,30 @@ public function walletLogin(Request $request): JsonResponse
         return response()->json([], Response::HTTP_UNPROCESSABLE_ENTITY);
     }
 
+    public function swashRegister(Request $request): JsonResponse
+    {
+        try {
+            new WalletAddress('eth', $request->input('address'));
+        } catch (InvalidArgumentException $exception) {
+            throw new UnprocessableEntityHttpException('Invalid wallet address');
+        }
+        $user = User::fetchBySwashWalletAddress($request->input('address'));
+        
+        if (null === $user) {
+            DB::beginTransaction();
+            $address = new WalletAddress('eth', User::generateRandomETHWalletForSwash());
+
+            $user = User::registerWithWallet($address, false);
+            $user->swash_wallet_address = $request->input('address');
+            if (Config::isTrueOnly(Config::AUTO_CONFIRMATION_ENABLED)) {
+                $this->confirmAdmin($user);
+            }
+            $user->saveOrFail();
+            DB::commit();
+        }
+        return response()->json(['sAd-id' => $user->wallet_address->toString()]);
```

Comment:
I think it should be `sAdId`. In you test you check `sAdId`

Reply:
Php automatically converts it to camel case style. 
However, I removed the dash and changed it to camel case style.